code/modules/mob/living/carbon/human/species/machine.dm

```diff
@@ -161,3 +161,33 @@
 			H.change_hair(new_style, 1)							// The 1 is to enable custom sprites
 		if(new_color)
 			H.change_hair_color(new_color)
+
+/datum/species/machine/spec_electrocute_act(mob/living/carbon/human/H, shock_damage, source, siemens_coeff, flags)
+	H.adjustBrainLoss(shock_damage)
+	H.adjust_nutrition(shock_damage)
+
+/datum/species/machine/handle_mutations_and_radiation(mob/living/carbon/human/H)
+	H.adjustBrainLoss(H.radiation / 100)
+	H.AdjustHallucinate(H.radiation)
+	H.radiation = 0
+	return TRUE
+
+/datum/species/machine/handle_brain_death(mob/living/carbon/human/H)
+	H.Weaken(60 SECONDS)
+	H.adjustBrainLoss(1) // 40 seconds to live
+	if(prob(20))
+		var/list/error_messages = list("Error 196: motor functions failing.",
```

Comment:
Here are some more messages for variety:

Error while displaying the error message.
Error 6344: Cannot delete file ~/²tmp⅛³³sy%³²¼ - No space left on device